README.md

```diff
@@ -0,0 +1,176 @@
+# react-smart-tabs
+> A simple to use tab system built with react.
+
+[![NPM Version][npm-image]][npm-url] [![Downloads Stats][npm-downloads]][npm-url]
+
+This Tab system was inspired by Chrome's tab functionality. It's very intuitive and works as you'd expect.
+
+You can use them as static tabs that just display what you want, or you can allow the user to create new tabs and/or close them. You can also choose if reordering the tabs via drag-and-drop is something you'd like.
+
+Finally, all state persists while the tabs are mounted, so you can change tabs without losing input values. All this logic is already implemented in the package, so you don't need to worry about it. It's meant to be simple to use.
+
+![](readme-demo.gif)
+
+
+## Installation
+
+```sh
+npm install --save react-smart-tabs
+```
+
+
+## Usage example
+
+Basically, you call *TabBar* passing it these props, depending on what you want the behavior to be:
+ - *newTab=*{functionThatReturnsTab} - Pass this if you want to allow the user to create new tabs.
+- *reorderable* - Pass this if you want the user to reorder the tabs via drag-and-drop
+- *closeable* - Pass this if you want the user to be able to close tabs.
+
+Then, as children of *TabBar*, you can call however many <Tab> components as you want, including none at all.
+
+Each *Tab* component can receive up to 3 props:
+  - *id* - Use a UUID or something similar, so you can diferentiate between them.
+  - *text* - This is the title text that will display at the top, on the TabBar
+  - *active* - Pass this to the one tab you want to be active onLoad.
+
+Finally, as children of the *Tab* components, you can pass the page you want to render. These can be multiple children, or a single component. You can even nest another TabBar inside it.
+
+Here are some examples of how to use it:
+
+```JavaScript
```

Comment:
do a simpler example

Reply:
I updated it to also include a very basic example which is displayed first. I've also kept the longer example. They're both in a collapsible drawer.